enforce-signer.tf

```diff
@@ -24,6 +24,11 @@ resource "google_project_iam_custom_role" "chainguard_signer_ca_role" {
     "privateca.certificateAuthorities.get",
     "privateca.certificates.create",
     "cloudkms.cryptoKeyEncrypterDecrypter",
+    "cloudkms.cryptoKeys.get", // This role and the following roles are needed for signing with a KMS-based CA
+    "cloudkms.cryptoKeys.list",
+    "cloudkms.cryptoKeyVersions.viewPublicKey",
+    "cloudkms.cryptoKeyVersions.useToSign",
+    "cloudkms.cryptoKeyVersions.list",
```

Comment:
Question: don't we use 

```
cloudkms.cryptoKeyVersions.useToDecrypt
cloudkms.cryptoKeyVersions.useToEncrypt
```
?

Reply:
I say that based on this perms for the role https://cloud.google.com/kms/docs/reference/permissions-and-roles#cloudkms.cryptoKeyEncrypterDecrypter

Reply:
👍🏻